webapp/templates/rewriterule-create.xhtml

```diff
@@ -0,0 +1,157 @@
+                            </select>
+                        </div>
+                        <div class="form-group col-sm-9">
+                            <label>Pattern</label>
+                            <input type="text" class="form-control requestPattern" value="" placeholder="Regular expression"/>
+                        </div>
+                        <div class="row">
+                            <h4 class="col-sm-12">Take the following action...</h4>
+                        </div>
+                        <div class="form-group col-sm-3">
+                            <label>Action</label>
+                            <select class="form-control actionType">
+                                <optgroup label="Redirects">
+                                    <option value="calli:canonical">Canonical (301)</option>
+                                    <option value="calli:alternate" selected="selected">Alternate (302)</option>
+                                    <option value="calli:describedby">Described by (303)</option>
+                                    <option value="calli:resides">Resides (307)</option>
+                                    <option value="calli:moved">Moved (308)</option>
+                                </optgroup>
+                                <optgroup label="GET requests">
+                                    <option value="calli:copy">Copy (200)</option>
+                                    <option value="calli:missing">Missing (404)</option>
+                                    <option value="calli:gone">Gone (410)</option>
+                                    <option value="calli:illegal">Illegal (451)</option>
+                                </optgroup>
+                                <optgroup label="Other requests">
+                                    <option value="calli:post">POST</option>
+                                    <option value="calli:put">PUT</option>
+                                    <option value="calli:patch">PATCH</option>
+                                    <option value="calli:delete">DELETE</option>
+                                </optgroup>
+                            </select>
+                        </div>
+                        <div class="form-group col-sm-9">
+                            <label>Target template</label>
+                            <div>
+                                <textarea class="pattern form-control purlTarget" rows="1"></textarea>
+                            </div>
+                        </div>
+                    </div>
+                </div>            
+            </div>            
+        </div>
+
+        <fieldset>
+            <div class="form-group row">
+                <label for="cache" class="col-sm-12">GET Cache-Control</label>
+                <div class="col-sm-3">
+                    <input type="text" id="cache" value="{calli:cacheControl}" placeholder="max-age=3600" class="form-control"
```

Comment:
Done for Purr, Proxy and RewriteRule - options can be created in all cases but if you match an existing option it replaces it with the "nice" label